apps/backend/src/app/modules/maps/map-credits.service.ts

```diff
@@ -78,10 +80,24 @@ export class MapCreditsService {
     if (!body.some((credit) => credit.type === MapCreditType.AUTHOR))
       throw new BadRequestException('Credits do not contain an AUTHOR');
 
+    const maxCredits = this.config.getOrThrow('limits.maxCreditsExceptTesters');
```

Comment:
Cache value in constructor?

Reply:
We do that for some small services like `S3Service` but in most parts of our code we don't, and I'd rather keep consistency.

The Nest `ConfigService.getOrThrow` is just a tiny wrapper on top of an Object using `'limits.maxCreditsExceptTesters'` directly as a key. Maaaybe there's some small speedup having it stored on the Object underlying the `MapCreditService` since iirc V8 does some memory optimizations for classes but it's probably so minuscule I don't think it matters. Ultimately care more about consistency and code-cleanliness here.